pages/index.js

```diff
@@ -1,7 +1,106 @@
-export default function HomePage() {
-	return (
-		<>
-			<h1>My Cosmetics Store</h1>
-		</>
-	);
+import styled from "styled-components";
+
+import { Btn } from "@/components/Button";
+
+import React, { useState } from "react";
+
+import myComponent from "@/components/myComponent1/myComponent.js";
```

Comment:
Components need to start with UpperCase. It will need to be changes in the import, wherever the component is called and in the component file.
```suggestion
import MyComponent from "@/components/myComponent1/myComponent.js";
```